proxy/client.go

```diff
@@ -28,7 +28,10 @@ var (
 	}
 	directClient = &http.Client{
 		Transport: directTransport,
-		Timeout:   15 * time.Second,
+		// TODO: with 15 i get timeouts for many Trakt and TMDB,
```

Comment:
`http.Client{Timeout}` is a general timeout to complete whole request.

I think we need to use `net.Dialer.Timeout` in our case, to make sure we do not wait for ever when meeting firewall or neetwork issues.

https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts